### ISSUE TITLE
I get error that says I need to define the access_token and access_token_secret when using user auth

### ISSUE THREAD

**Issue by danieljimenez0255** [HunterLarco/twitter-v2#18]
So I am building  react application that allows users to login using twitter auth from firebase and once they login it displays their user timeline. So I am trying to use this npm package to be able to give me the user's twitter username which I will pass into another component to retrieve the user timeline to be  displayed. 

I made sure to install the twitter-v2 package, and I set up my client variable with all the needed keys. Right now I am trying to test it out to see if it works by retrieveing tweets with a tweets id:
![github_twitterv2issue](https://user-images.githubusercontent.com/42664403/98493107-80193200-21ee-11eb-9e00-060effa96c98.png)

But then I get this weird issue:
![github_twitterv2issue2](https://user-images.githubusercontent.com/42664403/98493132-8dceb780-21ee-11eb-9818-9dada0d265da.png)

I also made sure this app is inside a approved project allowed to use the v2 endpoint and I have the correct keys to be able to pass in. I was wondering do you guys know how I would be able to fix this issue? Thank you all for the help I really appreciate it!

**Comment by HunterLarco** [comment]
This error is shown when the client hasn't been correctly constructed with Twitter's required API keys. Could you share the code where you construct the client so that we can dig deeper?

**Comment by danieljimenez0255** [comment]
![twitterV2installImg](https://user-images.githubusercontent.com/42664403/98880016-a5e74680-243b-11eb-80c2-44f5e5401af2.png)
The image above shows how I imported the twitter v2 package into the React Component. And below shows how I set the client 

![clientSetUp](https://user-images.githubusercontent.com/42664403/98880069-be576100-243b-11eb-8531-ea42a82aba3b.png)

Then I'd place the appropriate keys where I put those placeholder values for each key. And then I called the get request for the tweets and it led to that error I got. Is the way I am setting things in the client wrong, possibly because of the way I'm importing the package or is it something else?
